docs/02-admin/01-installation/docker.md

```diff
@@ -134,60 +127,110 @@ OAUTH_PASSPHRASE=<Your (optional) passphrase from above here>
+In this case you need to update the `compose.prod.override.yml` file with:
+
+```yaml
+services:
+  www:
+    image: "ghcr.io/mbinorg/mbin:latest-caddy"
+    pull_policy: never
+  php:
+    image: "ghcr.io/mbinorg/mbin:latest"
+    pull_policy: never
+  messenger:
+    image: "ghcr.io/mbinorg/mbin:latest"
+    pull_policy: never
+```
 
-The filesystem ACL is disabled by default, in the `mbin` image. You can set the environment variable `ENABLE_ACL=1` to enable it. Remember that not all filesystems support ACL. This will cause an error if you enable filesystem ACL for such filesystems.
+> ![NOTE]
+> You can replace `latest` with a version number e.g `1.0.0`
 
-## Run Production
+### Running the containers
 
-If you created the file `compose.override.yml` with your configs (`cp compose.prod.yml compose.override.yml`), running production would be the same command:
+Run the services in the background (`-d` means detach, but this can also be omitted for testing or debugging purposes):
 
 ```bash
+# Go to the docker directory within the git repo
+cd docker
+
+# Starts the containers
 docker compose up -d
 ```
 
-**Important:** The docker instance is can be reached at [http://127.0.0.1:8008](http://127.0.0.1:8008), we strongly advise you to put a reverse proxy (like Nginx) in front of the docker instance. Nginx can could listen on ports 80 and 443 and Nginx should handle SSL/TLS offloading. See also Nginx example below.
+See your running services via: `docker compose ps`.
+
+Then, you should be able to access the new instance via [http://localhost](http://localhost:8008).
+You can also access RabbitMQ management UI via [http://localhost:15672](http://localhost:15672).
+
+## Notes
 
-If you want to deploy your app on a cluster of machines, you can
-use [Docker Swarm](https://docs.docker.com/engine/swarm/stack-deploy/), which is compatible with the provided Compose
-files.
+### Uploaded media files
+
+Uploaded media files (e.g. photos uploaded by users) will be stored on the host directory `storage/mbin/public/media`
+by the `php` container and served by the Caddy web server in the `www` container as static files.
+
+Make sure `KBIN_STORAGE_URL` in your `.env` configuration file is set to be `https://yourdomain.tld/media`
+(assuming you setup Nginx with SSL certificate by now).
+
+You can also serve those media files on another server by mirroring the files at `storage/mbin/public/media` and
+changing `KBIN_STORAGE_URL` correspondingly.
+
+### Filesystem ACL support
+
+The filesystem ACL is disabled by default, in the `mbin` image. You can set the environment variable `ENABLE_ACL=1` to
+enable it. Remember that not all filesystems support ACL. This will cause an error if you enable filesystem ACL for such
+filesystems.
 
 ### Mbin NGINX Server Block
 
+> ![WARNING]
+> This is not up to date! PRs for a valid nginx service and config welcome!
```

Comment:
Why would this not be up to date?

Reply:
Caddy was pulled out of the `php` image, so the config is not up to date to use `php-fpm`. I haven't tested whether it still works and this config uses HTTPS, so I don't have a domain to test it with.

Reply:
Nginx is a reverse proxy that does SSL offloading, that means in normal terms: Nginx will provide HTTPS and will remove the SSL and connect to the service (without the need of SSL) and then it will add back the SSL to the response. So, Nginx can call the www (caddy) port without the need of SSL without any issues.

Reply:
Removed the warning.